libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts

```diff
@@ -188,7 +188,145 @@ export function getCompletionSnippets(range: monacoTypes.IRange, monaco): monaco
       insertText: '// SPDX-License-Identifier: MIT',
       insertTextRules: monaco.languages.CompletionItemInsertTextRule.InsertAsSnippet,
       range
+    },
+    /* eslint-disable */
+    // Related to https://github.com/juanfranblanco/vscode-solidity/blob/master/snippets/solidity.json
+    {
+      label: 'erc20i',
```

Comment:
probably the labels should be more explicit, like `generate erc20i` for instance.

Reply:
> probably the labels should be more explicit, like `generate erc20i` for instance.

that is a keyword to show the list

Reply:
> probably the labels should be more explicit, like `generate erc20i` for instance.

that is a keyword to show the list

Reply:
> btw just wondering, do we already have snippet for generating loops?

yes, we have

Reply:
> that is a keyword to show the list. We can update the detail field

yes, in  https://github.com/juanfranblanco/vscode-solidity/raw/master/screenshots/ercautocomplete1.png
there's a quick and a full description associated with the keyword.

Reply:
> > probably the labels should be more explicit, like `generate erc20i` for instance.
> 
> that is a keyword to show the list. We can update the detail field

updated @yann300